src/customers/queries.ts

```diff
@@ -50,7 +50,7 @@ export const customerDetails = gql`
       privateMetadata @include(if: $PERMISSION_MANAGE_STAFF) {
         ...MetadataItem
       }
-      orders(last: 5) @include(if: $PERMISSION_MANAGE_ORDERS) {
+      orders(first: 5) @include(if: $PERMISSION_MANAGE_ORDERS) {
```

Comment:
question: should't stay `last` as you want recent orders?

Reply:
Well, that's what was in the code already but it seems like there was undocumented change in Saleor API reversing the order (and no one noticed). If you go to master staging right now where there is still `(last:5)` you'll get the oldest orders. There's no sort on GraphQL here either. I'll write make a ticket so that we can explicitly sort orders here.